usaspending_api/spending_explorer/v2/filters/spending_filter.py

```diff
@@ -64,10 +64,7 @@ def spending_filter(alt_set, queryset, filters, _type):
 
             # recipient
             elif key == "recipient":
-                alt_set = alt_set.filter(
-                    Q(award__latest_transaction__assistance_data__awardee_or_recipient_legal=value)
-                    | Q(award__latest_transaction__contract_data__awardee_or_recipient_legal=value)
-                )
+                alt_set = alt_set.filter(Q(award__latest_transaction_search__recipient_name_raw=value))
```

Comment:
I believe the existing columns (`awardee_or_recipient_legal`), on the two tables, are currently mapped to `recipient_name` not `recipient_name_raw`

Reply:
It looks like you're correct. I just used `recipient_name_raw` since that's the column that the ticket mentioned. I'll update this query to use recipient name.

This is from the FPDS to Transaction Search mappings:
```jsonc
# transaction_fpds col name : transaction_search col name
"awardee_or_recipient_legal": "recipient_name
```

Reply:
Updated the recipient column

Reply:
Thanks Andy! I didn't see that the ticket mentioned the `_raw` field.